dev-libs/libsass/libsass-3.6.3.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools multilib-minimal
+
+if [[ ${PV} = *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/sass/libsass.git"
+	inherit git-r3
+	KEYWORDS=
+else
+	SRC_URI="https://github.com/sass/libsass/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux"
+fi
+
+DESCRIPTION="A C/C++ implementation of a Sass CSS compiler"
+HOMEPAGE="https://github.com/sass/libsass"
+LICENSE="MIT"
```

Comment:
Well actually I checked upstream and they have some custom license which looks pretty general. Could you check it's actually MIT? Fastly glancing through it looked like so, but to be sure...

Reply:
it says this all the way at the bottom of the README.md:

> ## Licensing
>
> Our MIT license is designed to be as simple and liberal as possible.

Reply:
Yep, as I said it looked similar but couldn't be 100 % sure. Thanks for checking.